src/ChatInput/ChatInput.tsx

```diff
@@ -62,11 +62,17 @@ export const ChatInput = forwardRef<ChatInputRef, ChatInputProps>(({
   stopIcon = <StopIcon />,
   attachIcon
 }, ref) => {
-  const { theme, isLoading, disabled, sendMessage, stopMessage, fileUpload } =
+  const { theme, isLoading, disabled, sendMessage, stopMessage, fileUpload, activeSessionId } =
     useContext(ChatContext);
   const [message, setMessage] = useState<string>('');
   const inputRef = useRef<HTMLTextAreaElement | null>(null);
 
+  useEffect(() => {
+    if(inputRef && inputRef.current) {
```

Comment:
`inputRef` is a ref so it will always be there - so i don't think you need the `if(inputRef` part.

Reply:
will it? it could be `null` but i guess you could simplify with `inputRef?.current`